span.go

```diff
@@ -60,21 +73,38 @@ func (e *traceExporter) convertSpan(s *trace.SpanData) *ddSpan {
 	if s.ParentSpanID != (trace.SpanID{}) {
 		span.ParentID = binary.BigEndian.Uint64(s.ParentSpanID[:])
 	}
+
+	httpCode := httpCode(s.Status.Code)
 	switch s.SpanKind {
 	case trace.SpanKindClient:
 		span.Type = "client"
+		if httpCode >= 400 && httpCode < 500 {
+			span.Error = 1
+		}
 	case trace.SpanKindServer:
 		span.Type = "server"
+		if httpCode >= 500 && httpCode < 600 {
```

Comment:
I think you can remove this condition and just use `fallthrough`, right?

Reply:
yep